### PR TITLE
Have travis deploy the site

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

**Comment by patcon** [comment]
THE DOUBLE PERIOD! IT EXISTS!

**Comment by patcon** [comment]
Works and tested! Ready for review and merge :)

File: .travis.yml [hunk 1]
```diff
@@ -2,18 +2,35 @@ language: ruby
 rvm:
   - 2.4.3
 
+before_install:
+  - openssl aes-256-cbc -K $encrypted_a3d58fa41b45_key -iv $encrypted_a3d58fa41b45_iv -in scripts/id_rsa.enc -out scripts/id_rsa -d
```
Comment (benhylau) [inline .travis.yml]: @patcon Can you explain how this works and what are `id_rsa.enc` and `encrypted_a3d58fa41b45_key`?

File: scripts/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+#
+USER="${1:-travis}"
+SERVER="${2:-ournetworks.ca}"
+IDENTITY_FILE="${3:-scripts/id_rsa}"
+SITE_DIR="/var/www/html"
+
+# This is done to skip the fingerprint check on new connection
+# - Skip the host fingerprint check
+# - Don't save the fingerprint to known_hosts
+scp -r \
```
Comment (dcwalk) [inline scripts/deploy.sh:18]: aaaaaaah, it should have been `scp`! 😂

File: .travis.yml (hunk 1 above)
Comment (dcwalk) [reply .travis.yml]: Is $encrypted_a3d58fa41b45_key a Travis CI env var?

**Review by dcwalk — COMMENTED** [review]
This looks great! So stoked, I think maybe we want to clarify how you are using the openssl in before_install. 💃

File: .travis.yml (hunk 1 above)
Comment (patcon) [reply .travis.yml]: It's what travis walks you through the setup of, after using the `travis encrypt-file` command. I'll link the docs for clarity :)

**Comment by patcon** [comment]
Made some tweaks to args to make the whole thing more transparent from travis config file. Ready for merge, if it looks good to y'all

**Review by dcwalk — COMMENTED** [review]
This is great! I can't approve b/c/ I'm also a PR owner, @garrying can you take a look.

DOUBLE DOTS

**Comment by benhylau** [comment]
Thanks for linking to travis docs re: key storage @patcon :)

**Review by benhylau — APPROVED** [review]

**Review by garrying — APPROVED** [review]
@patcon thanks! i'm gonna merge this

**Pull request merged**